tests/tests.py

```diff
@@ -154,9 +154,11 @@ def test_fields(self):
         self.assertGreater(len(fields), 10)
 
 
-class MyPermissionsTests(JiraTestCase):
+class MyPermissionsServerTests(JiraTestCase):
     def setUp(self):
-        JiraTestCase.setUp(self)
+        super().setUp()
+        if self.jira._is_cloud:
+            self.skipTest("server only test class")
```

Comment:
```suggestion
```

Not required as we use a pytest mark to determine what runs on the cloud, see below.

This opt-in method hopefully makes the tests easier to understand, let me know if that is not the case !

Reply:
Without the ```skipTest```, the test class will run for the server and I think it fails with 400 bcs the server does not know what to do what the new ```permissions``` parameter added for cloud.

Reply:
Yep but for this `MyPermissionsServerTests` we don't need the skip (as the cloud tests will never run it anyway as it does not have the marker)

Reply:
Yep you're right ! I forgot that only tests marked with ```allow_on_cloud``` will run on the cloud. In that case that check is redundant .